### PR TITLE
Update influxdb and grafana to newer versions

### LINKED ISSUE #1698: Unable to build grafana for other platforms
After #1624 is merged I'm not able to build grafana for other platforms than `amd64`.

See https://github.com/kubernetes/heapster/pull/1624/files#r123724923

### PR DESCRIPTION
**Pull request by luxas**
Updates grafana to v4.3.2 and influxdb to v1.3.0

fixes https://github.com/kubernetes/heapster/issues/1698 as a consequence

/cc @piosz @DirectXMan12 @mwielgus

### CODE REVIEW

**Comment by andyxning** [comment]
@luxas I have no idea why we need this for fix #1698. If you need to update grafana and influxdb, can you please split the update commits in another PR?  

We should make this PR suitable to just fix #1698 :)

**Comment by luxas** [comment]
@andyxning Why not update the others too?
Sure, it can be done separately, but they should be updated anyway so why not?

**Comment by andyxning** [comment]
@luxas No special tastes for me. Just to KISS. :)

If we update both, how about we update the description to doc the update. :)

**Comment by luxas** [comment]
Done

File: influxdb/Makefile [hunk 1]
```diff
@@ -69,7 +69,7 @@ push: gcr-login $(addprefix sub-push-,$(ALL_ARCHITECTURES))
 
 sub-push-%:
 	$(MAKE) ARCH=$* PREFIX=$(PREFIX) VERSION=$(VERSION) build
-	docker push $(PREFIX)/heapster-influxdb-$(ARCH):$(VERSION)
+	docker push $(PREFIX)/heapster-influxdb-$*:$(VERSION)
```
Comment (andyxning) [inline influxdb/Makefile]: @luxas Should `(ARCH)` be replaced with `*`?

**Comment by andyxning** [comment]
@piosz When this PR is merged, we should build and push `heapster-grafana-amd64:v4.3.2` and `heapster-influxdb-amd64:v1.3.0` to gcr.io.

File: influxdb/Makefile (hunk 1 above)
Comment (luxas) [inline influxdb/Makefile]: yes, this was an oversight from earlier.
`$*` resolves to the `%` parameter, which is the arch being built. `$(ARCH)` in this case resolves statically to amd64 every time

Comment (andyxning) [inline influxdb/Makefile]: Good catch! And glad to see that i have learned another info about `make`. 

The link below is just for any user interesting in this to consult: https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html

File: grafana/Makefile [hunk 2]
```diff
@@ -19,26 +19,25 @@
...
 # Set default base image dynamically for each arch
 ifeq ($(ARCH),amd64)
 	BASEIMAGE?=busybox
 	CC=gcc
 endif
-ifeq ($(ARCH),armhfrm)
+ifeq ($(ARCH),arm)
 	BASEIMAGE?=armhf/busybox
-	CC=arm-linux-gnueabi-gcc
+	CC=arm-linux-gnueabihf-gcc
```
Comment (andyxning) [inline grafana/Makefile]: I am not familiar with `armhf`, so i have no idea about this change. Others seem good to me.

Comment (luxas) [inline grafana/Makefile]: Basically the newer kube-cross image has only the armv7 variant (armhf) of gcc.
Earlier versions of kube-cross came bundled with the armv5 (armel) variant of gcc

**Comment by andyxning** [comment]
@luxas @piosz We should also add new relesse note about this new version in `RELEASES.md`.

File: grafana/Makefile (hunk 2 above)
Comment (andyxning) [inline grafana/Makefile]: sgtm

**Comment by luxas** [comment]
ping @piosz @DirectXMan12 @mwielgus Please merge and push new images

**Comment by luxas** [comment]
ping @piosz @DirectXMan12 @mwielgus

**Comment by andyxning** [comment]
@luxas Please resolve the conflicts. :)

**Comment by miry** [comment]
@luxas in the merged PR we have grafana v4.4.1, I think you can use the latest versions:

Grafana https://github.com/grafana/grafana/releases/tag/v4.4.3
influxbd v1.3.3 https://github.com/influxdata/influxdb/releases/tag/v1.3.3

**Comment by DirectXMan12** [comment]
can we close this because of the new PR?

**Comment by luxas** [comment]
Yes, closing

**Pull request closed without merging**